addons/hr/models/hr_employee.py

```diff
@@ -597,11 +597,11 @@ def _get_calendar_attendances(self, date_from, date_to):
 
     def _get_marital_status_selection(self):
         return [
-            ('single', 'Single'),
-            ('married', 'Married'),
-            ('cohabitant', 'Legal Cohabitant'),
-            ('widower', 'Widower'),
-            ('divorced', 'Divorced')
+            ('single', _('Single')),
+            ('married', _('Married')),
+            ('cohabitant', _('Legal Cohabitant')),
+            ('widower', _('Widower')),
+            ('divorced', _('Divorced'))
```

Comment:
```suggestion
            ('divorced', _('Divorced')),
            ('concubine', _('Concubine'))
```
What about this? 🤔

Reply:
Hello @MohammedBasioni @MiquelRForgeFlow @ticodoo ,
I wouldn't do the functional addition as those status aren't international legal concepts. If additional statuses are seen as necessary, the addition will probably be done in the relative l10n rather than in standard, but we would do it in a separate task anyway.
However I think that the addition of the trailing comma is welcome :slightly_smiling_face: 
Thanks!

Reply:
Yeah, it was a bit on the joke-side, but I like the l10n solution. I forgot to add https://en.wikipedia.org/wiki/Civil_union, which a lot of countries had already.